pymeasure/experiment/parameters.py

```diff
@@ -138,6 +138,12 @@ def value(self):
 
     @value.setter
     def value(self, value):
+        if isinstance(value, str):
+            if value.lower() in "true":
+                value = True
+            elif value.lower() == "false":
+                value = False
+
```

Comment:
Shouldn't there be an `else` clause that errors out? What kind of `value` strings do you want to accept here? I think as implemented (case-insensitive "true"/"false") is reasonable. As it is, i think a value of `"0"` or `"blalblubb"` would be stored as `True`, which is probably not intended.

Reply:
I didn't implement the `else` clause because I thought to leave as much of the original behavior as possible; there is specifically a test (tests\experiment\test_parameters.py::test_boolean_value) that tests for a `BooleanParameter` accepting a string ('a' in this case) and expects it to result in `True`.
I do agree that it makes no sense for the `BooleanParameter` to accept other strings, so do you think I should modify this test-case?

Reply:
hmm, I lean towards changing the expected/tested behaviour. Can you find out who implemented that test? (probably @cjermain  :D)

Reply:
> leave as much of the original behavior as possible

good call on that decision, by the way! In principle the right way to go. As we are still < 1.0 (but around for quite some time), maybe we should start collecting behaviour that we want to change/break before 1.0, and have one release later that implements all the breaks in one go, with good documentation.

Reply:
> Can you find out who implemented that test?

Indeed, as far is I can find it was implemented by @cjermain, so maybe he can comment on this behavior.